cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListConfigCurrency.cy.js

```diff
@@ -0,0 +1,81 @@
+import apiKeyCreateBody from "../../fixtures/create-api-key-body.json";
+import createConnectorBody from "../../fixtures/create-connector-body.json";
+import getConnectorDetails from "../PaymentMethodListUtils/utils";
+import merchantCreateBody from "../../fixtures/merchant-create-body.json";
```

Comment:
I guess it is better to until https://github.com/juspay/hyperswitch/pull/5180 is merged as it helps reduce complexities.